tests/test_scripts.py

```diff
@@ -2777,6 +2776,47 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting_load_gzipped_bigcount():
+    infile = utils.get_temp_filename('test_ct')
+    ct = khmer.new_counting_hash(10, 1e7, 4)
+    ct.set_use_bigcount(True)
+    for i in range(500):
+        ct.count('ATATATATAT')
+    ct.save(infile)
+    outfile = infile + '.gz'
+    with open(infile, 'rb') as f_in:
```

Comment:
When I use these same line, it works with me and fails with Jenkins. I will figure this out.